### PR TITLE
Email signup confirmation

### PR DESCRIPTION
**Pull request by BjjMeche**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -75,4 +75,8 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (pmmenzel) [inline config/environments/production.rb]: `host` needs to be herokuapp address, not `localhost`

Comment (BjjMeche) [inline config/environments/production.rb]: host left localhost, currently we do not have heroku set up for party planner

Comment (pmmenzel) [inline config/environments/production.rb]: this is a test

Comment (pmmenzel) [inline config/environments/production.rb]: This is another test

**Pull request merged**